### PR TITLE
added timeouts to REXEC

### PR DESCRIPTION
**Pull request by pi-r-p**
added timeouts for REXEC and REXECZ requests.

### CODE REVIEW

File: warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java [hunk 1]
```diff
@@ -156,6 +169,8 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
       
     } catch (WarpScriptException e) {
       throw e;
+    } catch(SocketTimeoutException e) {
+      throw new WarpScriptException(getName() + " Timeout : check configurations " + WARPSCRIPT_REXEC_CONNECT_TIMEOUT + " and " + WARPSCRIPT_REXEC_READ_TIMEOUT);
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java]: Please remove space before ':'

File: warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java [hunk 2]
```diff
@@ -62,7 +70,10 @@ public REXEC(String name, boolean compress) {
     super(name);
   
     String patternConf = WarpConfig.getProperty(WARPSCRIPT_REXEC_ENDPOINT_PATTERNS, DEFAULT_ENDPOINT_PATTERNS);
-
+    String readTimeout = WarpConfig.getProperty(WARPSCRIPT_REXEC_READ_TIMEOUT,DEFAULT_REXEC_READ_TIMEOUT);
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java]: Please insert space after ','

File: warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java [hunk 3]
```diff
@@ -62,7 +70,10 @@ public REXEC(String name, boolean compress) {
...
+    this.connReadTimeout= Integer.parseInt(readTimeout);
+    String connectTimeout = WarpConfig.getProperty(WARPSCRIPT_REXEC_CONNECT_TIMEOUT,DEFAULT_REXEC_CONNECT_TIMEOUT);
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java]: Please insert space after ','

File: warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java (hunk 1 above)
Comment (pi-r-p) [reply warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java]: done

File: warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java (hunk 3 above)
Comment (pi-r-p) [reply warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java]: done

**Pull request merged**